### DISCUSSION TITLE
New logo!

### DISCUSSION THREAD

**Discussion by DrSpaldo** [immich-app/immich#8428] [General]
![download](https://github.com/immich-app/immich/assets/8526509/8f61d102-a078-493c-836f-763736d78b37)

Interesting new Immich logo

**Comment by aniruddhas** [comment]
Yes. 1st April. 
https://github.com/immich-app/immich/discussions/8430#discussion-6444971

**Reply by yydcool** [reply]
imao, it looks way better than the old one

**Comment by d1vanloon** [comment]
Graphic design is my passion
